go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openyurtio/openyurt
 
-go 1.20
```

Comment:
please don't upgrade golang version in this pull request.

Reply:
done